code/game/objects/structures/tables_racks.dm

```diff
@@ -591,7 +591,7 @@
 	verbs -= /obj/structure/table/verb/do_flip
 	typecache_can_hold = typecacheof(typecache_can_hold)
 	for(var/atom/movable/held in get_turf(src))
-		if(is_type_in_typecache(held, typecache_can_hold))
+		if(is_type_in_typecache(held, typecache_can_hold) && held.anchored == 0)
```

Comment:
```suggestion
		if(!held.anchored && held.move_resist != INFINITY && is_type_in_typecache(held, typecache_can_hold))
```
`move_resist` is also used to say an object can't be moved. Left over from a rework that never fully got finished

Reply:
I'd forgotten about move resist, this means no more terror empress surfing down corridors on a surgical table :(

Reply:
Those got `MOVE_FORCE_STRONG` as value. `INFINITY` is only reserved for things that should never ever move